en/chronos.rst

```diff
@@ -231,8 +231,16 @@ objects::
     echo $time->toRssString();       // Thu, 25 Dec 1975 14:15:16 -0500
     echo $time->toW3cString();       // 1975-12-25T14:15:16-05:00
 
-    // Get the quarter
-    echo $time->toQuarter();         // 4;
+    // Get the quarter/week
```

Comment:
Instead of just "week", may be use "week of the year", "week number" or "calendar week".